src/pydrex/velocity_gradients.py

```diff
@@ -0,0 +1,42 @@
+"""> PyDRex: Steady-state solutions of velocity gradients for various flows."""
+import functools as ft
+
+import numba as nb
+import numpy as np
+
+
+@nb.njit(fastmath=True)
+def _simple_shear_2d(x, direction=1, deformation_plane=0, strain_rate=1):
```

Comment:
variable `x` is not used; is that on purpose?

Reply:
Yes, intended, for now only uniform shear flow. The 'x' is required just to catch the position arg that is passed to the function inside `update_orientations` (which I designed so that it can take position-dependent velocity gradients at some point).